ui/main/src/app/modules/feed/components/map/map.component.scss

```diff
@@ -21,6 +21,9 @@
     border-radius: 10px;
```

Comment:
Could you just fix the copyright header:  Copyright (c) 2022-**2023**, Alliander